### PR TITLE
Add link to Internationalization concept page from i18n API articles

### LINKED ISSUE #9300: Content bug: WebExtensions i18n articles don't link to Internationalization page
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n/getMessage

## Specific page section or heading?

The introduction of both articles, and also the `messageName` param of the getMessage method.

## What is the problem?

The i18n documentation is very brief, and does not make any reference to the detailed tutorial about Internationalization / localization..

## What did you expect to see?

 There should be prominently featured links to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary
We have a concept article about internationalization, however, there is no mention of this page on the related i18n API pages. This change is to add a cross-reference.

#### Motivation
Wo ensure developers have easy access to relevant internationalization information.

#### Related issues
Resolves WebExtensions i18n articles don't link to Internationalization page [#9300](https://github.com/mdn/content/issues/9300)

#### Metadata
This PR…
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/internationalization/index.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ So, you've got your message strings set up, and your manifest. Now you just need
 - The {{WebExtAPIRef("i18n.getAcceptLanguages()")}} and {{WebExtAPIRef("i18n.getUILanguage()")}} methods could be used if you needed to customize the UI depending on the locale — perhaps you might want to show preferences specific to the users' preferred languages higher up in a prefs list, or display cultural information relevant only to a certain language, or format displayed dates appropriately according to the browser locale.
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/internationalization/index.md:159]: Why was this removed?

Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/internationalization/index.md:159]: @Rob--W Aside from fixing the formatting issue in line 162, I haven't removed anything here.

**Review by hamishwillee — APPROVED** [review]
This sorts out the problem as described. Rob's comment about removed content appears invalid so I am going to merge.

FWIW, IMO it would be good if there were "discrete" examples of usage in the method pages as well, with a link to the example code. Sometimes you just want a code fragment for a particular API in its own context.
